src/server/scripts/Northrend/FrozenHalls/HallsOfReflection/halls_of_reflection.cpp

```diff
@@ -107,7 +107,11 @@ class npc_hor_leader : public CreatureScript
             }
         }
 
-        SendGossipMenuFor(player, DEFAULT_GOSSIP_MESSAGE, creature->GetGUID());
+        if (creature->GetEntry() == NPC_SYLVANAS_PART1)
+            SendGossipMenuFor(player, 15215, creature->GetGUID());
+
+        if (creature->GetEntry() == NPC_JAINA_PART1)
+            SendGossipMenuFor(player, 15339, creature->GetGUID());
```

Comment:
please add brackets, also the conditions of these 2 `if` blocks will never be both true (either one of them is true or none), so I'd use `if-else` or `switch` instead of 2 `if`s

Reply:
could you give me an example of where the brackets should go, shin?  first time!

Reply:
@Voxstrasza done :)

Reply:
thank you for that, i'll keep it in mind for the next CPP i modify!